### DISCUSSION TITLE
login button click is not working

### DISCUSSION THREAD

**Discussion by Haritest** [Sunbird-Ed/Community#213] [Installation] [answered]
Hi Team,

 None of the buttons on the login screen is not working. And there are no logs in keycloak and player service. Kindly help with this.
We are using 4.3 version deployment

![awssb](https://user-images.githubusercontent.com/13552553/193835211-83aa04db-7181-47a7-8374-1748acf33537.png)

Thank you

**Comment by beepdot** [comment, marked as answer]
Are there any errors in the network / console tab in your browsers? If yes, please post the details.

**Comment by Haritest** [comment]
No logs in the network tab, Please find the logs in the console tab when clicking LOGIN button

*Uncaught TypeError: Cannot read properties of undefined (reading 'hashTagId')
    at logInteractEvent (telemetry_service.js?v=1.15.b:2596:26)
    at doLogin (telemetry_service.js?v=1.15.b:2622:5)
    at HTMLButtonElement.onclick (auth?client_id=portal&state=93237f83-55ef-46e0-ac26-26c5c48b9a0a&redirect_uri=https%3A%2F%2Fsb-aws.tk%2Fresources%3Fauth_callback%3D1&scope=openid&response_type=code&version=4:89:129)
logInteractEvent @ telemetry_service.js?v=1.15.b:2596
doLogin @ telemetry_service.js?v=1.15.b:2622
onclick @ auth?client_id=portal&state=93237f83-55ef-46e0-ac26-26c5c48b9a0a&redirect_uri=https%3A%2F%2Fsb-aws.tk%2Fresources%3Fauth_callback%3D1&scope=openid&response_type=code&version=4:89*

![image](https://user-images.githubusercontent.com/13552553/194235817-7867474f-52ee-4617-86fb-e847cc168149.png)

**Reply by beepdot** [reply]
Do you have an organisation named `ntp`? If not, please create it. If you are unable to understand my query, please read this thread - https://github.com/orgs/Sunbird-Ed/discussions/115 as we have already resolved similar issue.

**Reply by monojit18** [reply]
@keshavprasadms 
I am getting the same error and I did follow all the communication at the link: https://github.com/orgs/Sunbird-Ed/discussions/115
And could resolve majority of the issues.

I have NTP org created with channel=ntp and slug=ntp (DB extract below)
I have NTP org in ES as well (ES curl update below)

when I hit the following REST API, I am always getting 200 OK but **count = 0**
Can you please help/suggest, if I am still missing something?

========
http://<domain>/api/org/v1/search
{
  "request": {
    "filters": {
      "id": "<ntp org id>"
    }
  }
}

========
cqlsh> SELECT id,channel,slug from sunbird.organisation;

 id                  | channel           | slug
---------------------+-------------------+-------------------
 0136386105328599041 | ArcheologyChannel | archeologychannel
             ORG_001 |          ROOT_ORG |              null
 0136444871414824966 |          sunbird3 |          sunbird3
 0136443139879485445 |          sunbird1 |          sunbird1
 0136443121580359684 |          sunbird2 |          sunbird2
 0136445155885957127 |           sunbird |           sunbird
 0136435154364579842 |  SocialogyChannel |  socialogychannel
 0136445591068999680 |               ntp |               ntp

========
curl localhost:9200/_cat/indices?pretty
yellow open badgeassociations     gJZT-m4NTaGoXlJY6bwW1g 5 1  0 0   1.2kb   1.2kb
yellow open cbatch-assessment     PSQJ4ethQt2OtxFxdzZIpg 5 1  0 0   1.2kb   1.2kb
yellow open location              6O2pFF4aT7aDrnQ7_vuX7Q 5 1  6 0  62.7kb  62.7kb
yellow open course-batch          mGux_PuXQTic4C-9L5YtkQ 5 1  0 0   1.2kb   1.2kb
yellow open userfeed              6fxgbJuYRciorlrec5_rlg 5 1  0 0   1.2kb   1.2kb
yellow open certreg               pA3gyHWkTNmY8mQkqKUSdw 5 1  0 0   1.2kb   1.2kb
yellow open usercourses           XGALLbdcQriKuX8xpwoK8Q 5 1  0 0   1.2kb   1.2kb
yellow open searchindex           OeOGEBLfRoifhepALPLKxA 5 1  0 0   1.2kb   1.2kb
yellow open kp_audit_log_2022_38  C-g1-5tkR4KLj4mqoAxtwA 5 1 26 0 442.8kb 442.8kb
yellow open metrics               SssqfaiNS6-eI-qH1DxVYQ 5 1  0 0   1.2kb   1.2kb
yellow open usernotes             m3qdurNASkmxsIaftWcAdQ 5 1  0 0   1.2kb   1.2kb
yellow open dialcode              bmD1C5nYQ4-KZd6qrKYP9A 5 1  0 0   1.2kb   1.2kb
yellow open dialcodemetrics       VLkMLc-1R2y5XtG2yrcTdw 5 1  0 0   1.2kb   1.2kb
yellow open announcementtype      oo-B3xHxSlKEGou-GBrwuA 5 1  0 0   1.2kb   1.2kb
yellow open compositesearch       0jXKAPGXT2-X45bdxbCA-w 5 1  0 0   1.2kb   1.2kb
yellow open cbatch                ObkwmN4xTSOYySwSli3QVg 5 1  0 0   1.2kb   1.2kb
yellow open user_alias            HfUd2rSMQjW3ineuL2bbPA 5 1  6 0  88.7kb  88.7kb
yellow open user-courses          JLitTF_vQ62ITPgiwMFDxw 5 1  0 0   1.2kb   1.2kb
yellow open userprofilevisibility qJqH-gHUTv-99bjXJhbH-A 5 1  0 0   1.2kb   1.2kb
**yellow open org_alias             Tb2I3J6yRKaDaGv8CdAxsw 5 1  7 0  70.3kb  70.3kb**
yellow open cert-templates        J0Ti5QyfTwSIISyLQYnWjQ 5 1  0 0   1.2kb   1.2kb
yellow open announcement          PcXpZW7RTI-41kdWYiSs6Q 5 1  0 0   1.2kb   1.2kb
yellow open cbatchstats           adTuTLG7QfW95z0zGtF9NA 5 1  0 0   1.2kb   1.2kb

========
curl -sS -X GET "localhost:9200/org_alias/_search?pretty" -H 'Content-Type: application/json' -d '{
>     "query": {
>         "match" : {
>             "slug" : "ntp"
>         }
>     }
> }'
{
  "took" : 1,
  "timed_out" : false,
  "_shards" : {
    "total" : 5,
    "successful" : 5,
    "skipped" : 0,
    "failed" : 0
  },
  "hits" : {
    "total" : 1,
    "max_score" : 0.2876821,
    "hits" : [
      {
        "_index" : "org_alias",
        "_type" : "_doc",
        "_id" : "0136445591068999680",
        "_score" : 0.2876821,
        "_source" : {
          "id" : "0136445591068999680",
          "channel" : "ntp",
          "createdBy" : "ad1ba6b9-9864-41ef-972e-6add52b7ed66",
          "createdDate" : "2022-10-12 17:10:16:799+0000",
          "description" : "NTP Organisation for Sunbird",
          "orgName" : "NTP",
          "organisationType" : 5,
          "rootOrgId" : "0136445591068999680",
          "slug" : "ntp",
          "status" : 1,
          "isSSOEnabled" : false,
          "isTenant" : true,
          "orgLocation" : [ ],
          "identifier" : "0136445591068999680"
        }
      }
    ]
  }
}

========
curl -sS -X GET "localhost:9200/org_alias/_search?pretty" -H 'Content-Type: application/json' -d '{
>     "query": {
>         "match" : {
>             "slug" : "sunbird"
>         }
>     }
> }'
{
  "took" : 1,
  "timed_out" : false,
  "_shards" : {
    "total" : 5,
    "successful" : 5,
    "skipped" : 0,
    "failed" : 0
  },
  "hits" : {
    "total" : 1,
    "max_score" : 0.2876821,
    "hits" : [
      {
        "_index" : "org_alias",
        "_type" : "_doc",
        "_id" : "0136445155885957127",
        "_score" : 0.2876821,
        "_source" : {
          "id" : "0136445155885957127",
          "channel" : "sunbird",
          "createdBy" : "ad1ba6b9-9864-41ef-972e-6add52b7ed66",
          "createdDate" : "2022-10-12 15:50:20:493+0000",
          "description" : "Default Channel",
          "orgName" : "Sunbird",
          "organisationType" : 5,
          "rootOrgId" : "0136445155885957127",
          "slug" : "sunbird",
          "status" : 1,
          "isSSOEnabled" : false,
          "isTenant" : true,
          "orgLocation" : [ ],
          "identifier" : "0136445155885957127"
        }
      }
    ]
  }
}

**Reply by beepdot** [reply]
@monojit18 The indexing is incorrect. `org_alias` should be an index alias and not an index.
This requires some steps to reindex and update the index data. Can you please create a new thread so we can discuss your specific use case?

**Reply by monojit18** [reply]
Sure @keshavprasadms, I am going to create a new thread with this query
Thanks

**Reply by luckynvdu** [reply]
@monojit18 @Haritest If your query is resolved here then would you mark this thread as answered?
